orcid-core/src/main/resources/i18n/api_en.properties

```diff
@@ -61,7 +61,7 @@ apiError.9018.userMessage=The ORCID record is locked.
 apiError.9018.moreInfo=http://members.orcid.org/api/api-error-codes
 apiError.9019.userMessage=The put code in the URL did not match that in the body of the message.
 apiError.9019.moreInfo=http://members.orcid.org/api/api-error-codes
-apiError.9020.userMessage=The put code in the URL did not match that in the body of the message.
+apiError.9020.userMessage=Invalid incoming message.
```

Comment:
It's an improvement, but invalid incoming message isn't specific enough. How will a developer know that to fix? I suspect we need more then one of these.